### PR TITLE
chore(reports): migrate from `prodpublick8s` to `publick8s`

### PR DESCRIPTION
**Pull request by lemeurherve**
Ref: https://github.com/jenkins-infra/helpdesk/issues/3351

### CODE REVIEW

File: config/reports.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ ingress:
       hosts:
         - reports.jenkins.io
         - reports.jenkins-ci.org
```
Comment (lemeurherve) [inline config/reports.yaml:18]: Note: there is no associated record in the `jenkins-ci.org` DNS zone.

To be removed in a later PR?

File: clusters/plugin-health-scoring.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+  - name: plugin-health-scoring
+    namespace: plugin-health-scoring
+    chart: jenkins-infra/plugin-health-scoring
+    version: 2.1.5
+    needs:
+      - public-nginx-ingress/public-nginx-ingress # Required to expose the service
+    values:
+      - "../config/plugin-health-scoring.yaml"
+    secrets:
+      - "../secrets/config/plugin-health-scoring/secrets.yaml"
```
Comment (dduportal) [inline clusters/plugin-health-scoring.yaml:10]: Looks like an error (`git add .` is bad :troll: )

**Review by dduportal — CHANGES_REQUESTED** [review]
Accidental file commited, otherwise LGTM

File: clusters/plugin-health-scoring.yaml (hunk 2 above)
Comment (lemeurherve) [reply clusters/plugin-health-scoring.yaml:10]: That was a `git add clusters`, completely unrelated! 😛 

Good catch, that's from a POC trying to allow external contributor to be in charge of dependency PR review for specific services

**Review by dduportal — APPROVED** [review]

**Pull request merged**